hw2_function/fastq_filtrator.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[51]:
+
+
+#Counting GC content
+def GC_comp(seq):
```

Comment:
Все переменные в питоне (за исключением классов и глобальных констант) принято обозначать маленькими буквами, даже если это аббревиатуры
```suggestion
def gc_comp(seq):
```